network/http/detectors-http.tf

```
@@ -44,7 +44,7 @@ resource "signalfx_detector" "http_regex_matched" {
 
   program_text = <<-EOF
     signal = data('http.regex_matched', ${module.filter-tags.filter_custom})${var.http_regex_matched_aggregation_function}${var.http_regex_matched_transformation_function}.publish('signal')
-    detect(when(signal < 1)).publish('CRIT')
+    detect(when(signal < 1, lasting('${var.http_regex_matched_lasting_duration}'))).publish('CRIT')
```

please follow existing implementation which force seconds as unit to make it possible arithmetical operations inside terraform: https://github.com/claranet/terraform-signalfx-detectors/blob/master/cloud/aws/alb/detectors-alb.tf#L58

```suggestion
     detect(when(signal < 1, lasting(asting='${var.http_regex_matched_lasting_duration_seconds}s'))).publish('CRIT')
```